external/model-preparation-algorithm/mpa_tasks/apis/task.py

```diff
@@ -262,11 +259,43 @@ def _init_test_data_cfg(self, dataset: DatasetEntity) -> Union[Config, None]:
         """
         return None
 
-    def _init_recipe_hparam(self) -> dict:
-        """
-        initialize recipe hyperparamter as dict.
+    def _overwrite_parameters(self):
```

Comment:
Is this a replacement function for "_init_recipe_hparam"?
It seems to still remain in the classification task.

Reply:
yes, it's a replacement. Instead of having `_init_recipe_hparam` in each task, it's maintained by BaseTask now and inherited and overwritten per task if desired.